### ISSUE TITLE
Slurm will not mark non responsive node as down, if node is within ResumeTimeout

### ISSUE THREAD

**Issue by rexcsn** [aws/aws-parallelcluster#2146]
Please see [wiki page](https://github.com/aws/aws-parallelcluster/wiki/Slurm-Issues#slurm-will-not-mark-non-responsive-node-as-down-if-node-is-within-resumetimeout) for details regarding this issue

Root cause of https://github.com/aws/aws-parallelcluster/issues/2117

We will track any feedback/comments in this Github issue, thank you!

**Comment by demartinofra** [comment]
Here is the fix which should be included in Slurm 20.11 to address this issue: https://github.com/SchedMD/slurm/commit/9140c2a97a4c89785a2e4c3d747f498a384a5c7c

**Comment by keien** [comment]
will the next parallelcluster release have a version of slurm with the above fix?

**Comment by demartinofra** [comment]
Unfortunately Slurm 20.11 which contains the fix hasn't been released yet so we can't guarantee the very next release of ParallelCluster will have it included.

**Comment by demartinofra** [comment]
Slurm 20.11 got released but the following regression is preventing ParallelCluster from upgrading: https://bugs.schedmd.com/show_bug.cgi?id=10298

**Comment by tbugfinder** [comment]
looks like fixed:
```
This is fixed in 20.11.1:

https://github.com/SchedMD/slurm/commit/466c2878ba6ef3724f97a9a7c3c775c9992fb9dc
```

**Comment by demartinofra** [comment]
Upgrade to Slurm 20.11 has been merged in develop:
* https://github.com/aws/aws-parallelcluster-cookbook/pull/871
* https://github.com/aws/aws-parallelcluster-node/pull/278
* https://github.com/aws/aws-parallelcluster/pull/2459

**Comment by rexcsn** [comment]
pcluster v2.11.0 is released, which contains slurm v20.11.7 that will fix the issue.
Resolving the issue now, please let us know if you have any question, thanks!
